webpack.production.min.config.js

```diff
@@ -0,0 +1,43 @@
+var webpack = require('webpack');
+
+module.exports = {
+  devtool: '#eval-source-map',
```

Comment:
This still uses `eval`, which should probably be avoided in production (see #217), maybe this shhould go into another build, maybe `.debug.js`.